packages/docs/src/pages/guidelines/component-maturity.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Component maturity
+---
+
+## Draft
+- **Do not implement**
+- Documentation and guidance is incomplete
+- Testing and validating is incomplete
+- Could be buggy or have accessibility issues
+
+## Work in progress
+- **Implement with caution**
+- Documentation and guidance is complete
+- Testing and validating is in progress
+- Could be buggy or have accessibility issues
+
+## Ready
+- **Ready to implement**
+- Documentation and guidance is complete
+- Testing and validating is complete
```

Comment:
Let's change validating to validation on all three stops.

As a post-PR item, I think we could link these out to detailed explanations of what testing and validation look like. Same for accessibility.

Reply:
I think so too, what it means for each of those steps to be "done"
- Documentation
- Testing
- Validation

Reply:
We have documentation outlined here https://github.com/CMSgov/design-system/blob/sw-writing-docs-update/guides/WRITING-DOCUMENTATION.md#documentation-and-guidance

and testing and validation outlined here https://github.com/CMSgov/design-system/blob/sw-propose-new-item-issue-template/.github/ISSUE_TEMPLATE/component_validation_checklist.md

Both are PRs awaiting review